.storybook/preview.js

```diff
@@ -3,7 +3,7 @@ import { BaseStyles } from '../src/components/BaseStyles'
 import Theme from '../src/components/Theme'
 
 export const parameters = {
-  // layout: 'centered',
+  layout: 'centered',
```

Comment:
Don't think we should do this, because:

![Screenshot 2021-11-08 at 11 58 12](https://user-images.githubusercontent.com/14276144/140730374-2f4d1538-6291-4631-8379-20f589f3a2ce.png)

Went over some stories and some of them are not looking as expected.